concordia/signals/handlers.py

```diff
@@ -13,6 +14,14 @@
 ASSET_CHANNEL_LAYER = get_channel_layer()
 
 
+@receiver(user_logged_in)
+def clear_reservation_token(sender, user, request, **kwargs):
+    try:
```

Comment:
I don't think this matters too much but I believe this could also be written `request.session.pop("reservation_token")`

Reply:
Ah you're right. If the tests pass I'm not going to fiddle with it.

Reply:
Yeah, I wouldn't spend more time on it